source/documentation/monitoring_apps/metrics.md

```diff
@@ -2,94 +2,56 @@
 
 ## Metrics
 
-Cloud Foundry provides time series data, or metrics, for each instance of your PaaS app. You can receive, store and view this data in a monitoring system of your choice by either:
+Cloud Foundry provides time series data, or metrics, for each instance of your PaaS app. You can receive, store and view this data in a monitoring system of your choice by deploying one of the following apps:
 
-- using the [Prometheus](https://prometheus.io/) [external link] endpoint provided by the GOV.UK PaaS team
-- deploying the `paas-metric-exporter` app to push metrics data in [StatsD](https://github.com/etsy/statsd/wiki) [external link] format
+- the [`paas-prometheus-exporter`](https://github.com/alphagov/paas-prometheus-exporter)
+- the [`paas-metric-exporter`](https://github.com/alphagov/paas-metric-exporter) to push metrics data in the [StatsD](https://github.com/etsy/statsd/wiki) [external link] format
 
 You can also view all metrics in a one-off snapshot by installing the Cloud Foundry CLI [log cache plug-in](https://github.com/cloudfoundry/log-cache-cli#installing-plugin) [external link].
 
-### Use the Prometheus endpoint
+### PaaS Prometheus exporter app
 
-Prometheus uses the following API endpoints to request metrics from Cloud Foundry:
+To use the PaaS Prometheus exporter app, you deploy it as an app on PaaS. The current metrics supported by this app are:
```

Comment:
It now includes service metrics. Doesn't make sense top point to the main readme of the project in https://github.com/alphagov/paas-prometheus-exporter to refer to the metrics the application can offer? That way we do not need to update the documentation every time the application changes.

You can say something like:

```
To use the PaaS Prometheus exporter app, you deploy it as an app on PaaS. It can collect metrics from your applications (i.e. CPU, memory, disk, app crashes, request and response codes) and backing services that inject metrics in the platform. See the [`paas-prometheus-exporter`](https://github.com/alphagov/paas-prometheus-exporter) for more details.
```

 https://github.com/alphagov/paas-prometheus-exporter